File: 3.实用功能封装/3.2handlePhotos/ZHHandlePhotos/ViewController.m
//
//  ViewController.m
//  ZHHandlePhotos
//
//  Created by Hayder on 16/8/8.
//  Copyright © 2016年 wangzhenhai. All rights reserved.
//

#import "ViewController.h"
#import "ZHImageVIew.h"
#import "ZHSnipButton.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
//    ZHSnipButton *button = [[ZHSnipButton alloc] initWithFrame:CGRectMake(kScreenW/2 - 50, kScreenH/2 - 50, 100, 100)];
//    button.snipButtonSize = CGSizeMake(50, 50);
//    
//    [self.view addSubview:button];
    
    ZHImageVIew *imageview = [[ZHImageVIew alloc] initWithFrame:CGRectMake(0, 20, [UIScreen mainScreen].bounds.size.width,  [UIScreen mainScreen].bounds.size.height - 20)];
    imageview.image  = [UIImage imageNamed:@"1.jpg"];
    [self.view addSubview:imageview];
}

@end

File: Demo/Swift/语法/3.Swift3.0- 逻辑分支/3.Swift3.0- 逻辑分支/ViewController.swift
//
//  ViewController.swift
//  3.Swift3.0- 逻辑分支
//
//  Created by Hayder on 16/9/21.
//  Copyright © 2016年 wangzhenhai. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
    }
    
    //MARK: 三目
    func demo1(){
    
        let x = 10
        
        //三目的语法和OC一样
        x > 5 ? print("大") : print("小")
        
        //'()' 表示空执行
        x > 5 ? print("大") : ()
    }
    func demo() {
    
        let x = 10
        
        /**
            1.条件不需要()
            2.语句必须有{}
         
        */
        if x > 5{
            
            print("大");
        }else
        {
            print("小")
        }
    
    }

}

